sormas-e2e-tests/src/test/java/org/sormas/e2etests/helpers/WebDriverHelpers.java

```diff
@@ -332,6 +332,11 @@ public String getCheckedOptionFromHorizontalOptionGroup(By options) {
     return baseSteps.getDriver().findElement(options).findElement(SELECTED_RADIO_BUTTOn).getText();
   }
 
+  public String getCheckedOption(By options, By selected) {
```

Comment:
`getCheckedOptionFromHorizontalOptionGroup` please use the implemented method

Reply:
Done